sys/install.sh

```diff
@@ -32,6 +32,11 @@ while : ; do
 		shift
 		continue
 		;;
+	"--without-pull")
+		export WITHOUT_PULL=1
+		shift
+		continue
```

Comment:
```suggestion
```

Reply:
the continue is important if we do shift, to avoid edge cases.

Reply:
After the case theres a shift and the continue is implicit because it loops again

Reply:
yeah, sorry, you're in the right.